.storybook/components/ColorList/ColorList.module.css

```diff
@@ -1,7 +1,7 @@
 .color-list__label {
   width: fit-content;
-  padding: var(--eds-size-half);
-  border-radius: var(--eds-theme-box-button-border-radius);
+  padding: calc(var(--eds-size-half) / 16 * 1rem);
```

Comment:
Size tokens are now unit-less, to support importing from figma, AND to allow usage with tailwind to generate size utility classes (e.g., `m-*`, `p-*`, etc.)